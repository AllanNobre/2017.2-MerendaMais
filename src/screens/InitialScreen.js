import React,{Component} from 'react';
import { StyleSheet, Text, View, Image, TouchableOpacity } from 'react-native';
import { Actions } from 'react-native-router-flux';

const imageCapa = require('../images/capa.jpg');


export default class InitialScreen extends React.Component {

    render() {
        return (
            <Image style={styles.im} source = {imageCapa}>
                <View style={styles.capa}>
                    <View style={{flex: 1, alignItems:'center'}}>
                        <Text style={styles.txt}>MERENDA +</Text>
                    </View>

                    <View style={{flex: 1, alignItems:'center', justifyContent: 'center'}}>
                        <TouchableOpacity
                        style={styles.btn}
                        activeOpacity= {0.7}
                        onPress = {() => Actions.loginConselheiro()}>
                            <Text style={styles.txt_btn}>Login</Text>
                        </TouchableOpacity>

                        <TouchableOpacity
                        style={styles.btn1}
                        activeOpacity= {0.7}
                        onPress = {() => Actions.registerScreen()}>
                            <Text style={styles.txt_btn}>Cadastrar</Text>
                        </TouchableOpacity>
                    </View>
                </View>
            </Image>

        );
    }
}


const styles=StyleSheet.create({
    im: {
        flex:1,
        width: null
    },
    capa: {
        flex: 1,
        backgroundColor:'transparent',
        shadowOffset:{width:0,height:1},
        shadowOpacity:1
    },
    btn: {
        width: 320,
        height: 65,
        alignItems:'center',
        justifyContent: 'center',
        marginBottom:60,
        backgroundColor:'#FF9500',
        borderRadius:10

    },
    txt_btn: {
        fontSize:20,
        color:'white'
    },
    btn1: {
        width: 320,
        height: 65,
        alignItems:'center',
        justifyContent: 'center',
        backgroundColor:'#FF9500',
        borderRadius:10

    },
    txt: {
        fontSize:60,
        fontWeight:'bold',
        color:'white',
        marginTop:140
    }
});
