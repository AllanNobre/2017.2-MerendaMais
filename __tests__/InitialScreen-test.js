import 'react-native';
import React from 'react';
import InitialScreen from '../src/screens/InitialScreen';

import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <InitialScreen />
    ).toJSON();
  expect(tree).toMatchSnapshot();
});
