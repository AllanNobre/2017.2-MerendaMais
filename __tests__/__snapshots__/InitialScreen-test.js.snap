// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Image
  source={1}
  style={
    Object {
      "flex": 1,
      "width": null,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
        "shadowOffset": Object {
          "height": 1,
          "width": 0,
        },
        "shadowOpacity": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "white",
            "fontSize": 60,
            "fontWeight": "bold",
            "marginTop": 140,
          }
        }
      >
        MERENDA +
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        collapsable={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        nativeID={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#FF9500",
            "borderRadius": 10,
            "height": 65,
            "justifyContent": "center",
            "marginBottom": 60,
            "opacity": 1,
            "width": 320,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          disabled={false}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "white",
              "fontSize": 20,
            }
          }
        >
          Login
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        collapsable={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        nativeID={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#FF9500",
            "borderRadius": 10,
            "height": 65,
            "justifyContent": "center",
            "opacity": 1,
            "width": 320,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          disabled={false}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "white",
              "fontSize": 20,
            }
          }
        >
          Cadastrar
        </Text>
      </View>
    </View>
  </View>
</Image>
`;
